src/components/ContextBar.js

```diff
@@ -163,17 +168,18 @@ export const ContextBar = ({
                 hover: { ...contextBarStyles.hover }
               },
               [
-                div({ style: { textAlign: 'center', color: colors.dark(), fontSize: '0.8em' } }, 'Rate'),
```

Comment:
`0.x` is the convention in the codebase. Changing the convention requires justification, via PR comment or otherwise.

Reply:
I see your point - made adjustments.